src/main/resources/META-INF/plugin.xml

```diff
@@ -161,5 +161,9 @@ For really big files and slow grammars, there is an appreciable delay when displ
 	  <defaultLiveTemplatesProvider implementation="org.antlr.intellij.plugin.templates.ANTLRLiveTemplatesProvider"/>
 	  <liveTemplateContext implementation="org.antlr.intellij.plugin.templates.ANTLRGenericContext"/>
 	  <liveTemplateContext implementation="org.antlr.intellij.plugin.templates.OutsideRuleContext"/>
+      <projectConfigurable groupId="project"
```

Comment:
I believe it would be better to add this configuration panel as a child of the `Languages & Frameworks` menu to avoid cluttering the menu options. And maybe rename it to `ANTLR v4 default settings` or something like that?

Reply:
Done